TestRunBindMountPropagation fails locally (`mountpoint \"/\" doesn't have optional field neither of [shared:]"`)

### Description

On latest lima:

```
--- FAIL: TestRunBindMountPropagation (0.10s)
    container_run_mount_linux_test.go:561: Running test propagation case rshared
    container_run_mount_linux_test.go:595: assertion failed: res.ExitCode is not exitCode:
        Command:  /usr/local/bin/nerdctl --namespace=nerdctl-test run -d --privileged --name nerdctl-testrunbindmountpropagation-rshared --mount type=bind,src=/tmp/TestRunBindMountPropagation1494058263/001/rw3593147632,target=/mnt1,bind-propagation=rshared ghcr.io/stargz-containers/alpine:3.13-org top
        ExitCode: 1
        Error:    exit status 1
        Stdout:
        Stderr:   time="2024-09-03T13:55:12-07:00" level=fatal msg="failed to parse volume options (\"bind\", \"/tmp/TestRunBindMountPropagation1494058263/001/rw3593147632\", \"rshared,rbind\"): mountpoint \"/\" doesn't have optional field neither of [shared:]"
```

@AkihiroSuda since this (apparently) involve Lima, do you have an opinion on what is going on?

Thanks.

### Steps to reproduce the issue

_No response_

### Describe the results you received and expected

na

### What version of nerdctl are you using?

main

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

### Host information

_No response_

--- a/cmd/nerdctl/container/container_run_mount_linux_test.go
+++ b/cmd/nerdctl/container/container_run_mount_linux_test.go
@@ -500,6 +500,8 @@ func TestRunVolumeBindMode(t *testing.T) {
 }
 
 func TestRunBindMountPropagation(t *testing.T) {
+	t.Skip("This test is currently broken. See https://github.com/containerd/nerdctl/issues/3404")
+
 	tID := testutil.Identifier(t)
 
 	if !isRootfsShareableMount() {

